frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnField.scss

```diff
@@ -1,15 +1,12 @@
 .odc-multi-column-field {
   &__row {
-    display: flex;
-    flex-direction: row;
     position: relative;
     margin-right: var(--pf-global--spacer--lg);
     margin-bottom: var(--pf-global--spacer--lg);
     max-height: var(--pf-global--spacer--xl);
```

Comment:
Just saw that we have a max height per row. unsure why that is. @rohitkrai03 ?

Reply:
Ping @rohitkrai03 ^

Reply:
I don't exactly remember as to why I added this but I think it was something to do with the UX designs at the moment and rows being bigger than the designs.